github/pulls_reviewers.go

```diff
@@ -13,18 +13,22 @@ import (
 // RequestReviewers creates a review request for the provided GitHub users for the specified pull request.
```

Comment:
> for the provided GitHub users

This was true before the change, but we'll need to update docs now to describe the new API.